ReisTecnologia/abc
# Basic layout on lessons page and added spinner while list loads

Lessons page should now have a basic layout and while the list of lessons load it should have a spinner while loading.

How to test: run ntl dev 
go to localhost:8888
check layout
enable throttle to slow 3G
reload page and check if while loading the list the spinner appears
